command/csi_plugin_status.go

```diff
@@ -0,0 +1,154 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type CSIPluginStatusCommand struct {
+	Meta
+	length  int
+	verbose bool
+}
+
+func (c *CSIPluginStatusCommand) Help() string {
+	helpText := `
+Usage: nomad csi plugin status [options] <id>
+
+  Display status information about a CSI plugin. If no plugin id is given, a
+  list of all plugins will be displayed.
+
+General Options:
+
+  ` + generalOptionsUsage() + `
+
+Status Options:
+
+  -short
+    Display short output. Used only when a single job is being
```

Comment:
Looks like this help text has some copy-paste errors from jobs. (Here and in `Synopsis()` as well)